### PR TITLE
finalize 2.5.1

### PR DESCRIPTION
**Pull request by YamatoSecurity**
I would like to release the new rules with the new convertor so want to release this minor release with the memory improvement. It's already past Mothers Day in Japan but still Mothers Day in other parts of the world so I decided to make it a Mothers Day release. =)

### CODE REVIEW

**Review by hitenkoku — APPROVED** [review]
@YamatoSecurity Thanks for your pull request.
LGTM.

File: CHANGELOG-Japanese.md [hunk 1]
```diff
@@ -1,10 +1,6 @@
 # 変更点
 
-## 2.6.0 [2023/XX/XX] "XXX"
-
-**新機能:**
-
-- XXX
+## 2.5.1 [2023/05/04] "Mothers Day Release"
```
Comment (fukusuket) [inline CHANGELOG-Japanese.md:3]: There was a typo, could you please check it? :)
```suggestion
## 2.5.1 [2023/05/14] "Mothers Day Release"
```

**Review by fukusuket — CHANGES_REQUESTED** [review]

**Comment by YamatoSecurity** [comment]
@fukusuket Thanks for the typo fix!

**Review by fukusuket — APPROVED** [review]
LGTM!🚀

**Pull request merged**